appserver/admingui/core/src/main/resources/org/glassfish/admingui/core/Strings.properties

```
@@ -434,8 +434,8 @@ commonTasks.task.aquarium=Payara News
 commonTasks.task.aquarium.toolTip=News from the Payara Community
 commonTasks.task.aquariumLink=http://www.payara.co/blog
 
-commonTasks.task.support=Get Payara Support
-commonTasks.task.support.toolTip=Payara Support
+commonTasks.task.support=Discover Payara Enterprise
+commonTasks.task.support.toolTip=Discover Payara Enterprise
 commonTasks.task.supportLink=http://www.payara.co/support
```

We could probably do with updating this link while we're at it so that it doesn't need to redirect :)
```suggestion
commonTasks.task.supportLink=https://www.payara.fish/enterprise/
```

---

Done